cryptostock/account/tests/test_total_value.py

```diff
@@ -0,0 +1,21 @@
+import decimal
+
+import pytest
+from account.tests.factory import OfferFactory, SalesDashboardFactory
+
+
+@pytest.mark.parametrize(
+    "count,price,expected_total_value",
+    [
+        ("10", "10", decimal.Decimal("100")),
+        ("5.5555", "5.555555", decimal.Decimal("30.8639")),
+        ("1", "0.555555", decimal.Decimal("0.5556")),
+        ("1", "0.000001", decimal.Decimal("0.0001")),
```

Comment:
Add this test, please
```suggestion
        ("1", "0.000001", decimal.Decimal("0.0001")),
        ("0.0001", "0.000001", ...),
```

Reply:
done